bigint/bigint.mbti

```diff
@@ -0,0 +1,41 @@
+package moonbitlang/core/bigint
+
+// Values
+fn from_decimal_string(String) -> BigInt
+
+fn from_int(Int) -> BigInt
+
+fn from_int64(Int64) -> BigInt
+
+fn make_zero() -> BigInt
+
+// Types and methods
+type BigInt
+impl BigInt {
+  compare(Self, Self) -> Int
+  debug_write(Self, Buffer) -> Unit
+  deep_clone(Self) -> Self
```

Comment:
when `deep_clone` is needed given BigInt is pure?

Reply:
fixed